libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -245,7 +245,7 @@ void AP_Camera::send_feedback(mavlink_channel_t chan)
     }
 
     mavlink_msg_camera_feedback_send(chan, 
-        gps.time_epoch_usec(),
+                                     AP::gps().time_epoch_usec(),
```

Comment:
Why all the whitespace ?

Reply:
@amilcarlucas Lines it up correctly with the argument list!

Small improvement ;-)

Reply:
I would say it is a great improvement! But doing this line and leaving the next ones in the same indentation leaves it in a strange (and worse) state.

Reply:
OK, I found a third way and moved the `chan` argument down :-)